### DISCUSSION TITLE
Dedicated website for alpine.js (documentation, showcase, tutorials, screencasts etc.)

### DISCUSSION THREAD

**Discussion by joshuablum** [alpinejs/alpine#889] [3. Feature ideas]
Hey there everyone 👋

So I've been using alpine for quite some time now and I feel like it deserves a nice website for the documentation and everything else related to it, like:

- cool examples
- maybe even some tutorial
- or high quality screencasts

I am happy to contribute building and maintaining it since I've been looking to become actively involved in open source work and gaining more experience in this regard.

As for the domain [js.org](https://js.org/) offers subdomains for projects so [alpine.js.org](https://alpine.js.org/) surely looks nice, doesn't it.

Just thought I throw this idea in here for an open discussion and see if there is any interest in it.

Keep up the great work y'all! ✌🏻 

Cheers
Joshua

**Comment by HugoDF** [comment]
This has been raised a few times, ultimately the decision lies with @calebporzio.

There are a few community projects attempting to deliver some of the value you're looking for, namely:

- https://github.com/alpine-collective/awesome
- https://alpinetoolbox.com (examples, resources, tools)
- (my personal attempts) https://alpinejs.codewithhugo.com/ and the Alpine.js weekly newsletter archive https://buttondown.email/alpinejs/archive
- Ryan's blog has some cool Alpine content as well https://ryangjchandler.co.uk/

**Reply by joshuablum** [reply]
Thank you for the reply @HugoDF, some resources I did not know about!

As you said, @calebporzio is the one who has to decide. Let's wait for his answer 😁
